src/main/resources/alma/index-config.json

```diff
@@ -99,8 +99,12 @@
             "heldBy" : {
                "properties" : {
                   "label" : {
-                     "index": "false",
-                     "type" : "text"
+                     "type" : "text",
+                     "fields": {
+                        "raw": {
```

Comment:
This is the mapping for doc type `item`, but we need it for `resource`.

Reply:
Thx! I will remove `item` as we don't need it and it just make trouble.